.golangci-kal.yml

```diff
@@ -44,8 +44,6 @@ linters-settings:
         # optionalOrRequired:
         #   preferredOptionalMarker: optional | kubebuilder:validation:Optional # The preferred optional marker to use, fixes will suggest to use this marker. Defaults to `optional`.
         #   preferredRequiredMarker: required | kubebuilder:validation:Required # The preferred required marker to use, fixes will suggest to use this marker. Defaults to `required`.
-        # requiredFields:
-        #   pointerPolicy: Warn | SuggestFix # Defaults to `SuggestFix`. We want our required fields to not be pointers.
```

Comment:
I think best to leave the config "shadow" so that we can see what we could configure if we decided to change

Reply:
Fixed!